Tools/TestWebKitAPI/Tests/IPC/IPCSerialization.mm

```diff
@@ -859,9 +860,9 @@ static void destroyTempKeychain(SecKeychainRef keychainRef)
     contact.get().emailAddress = @"admin@webkit.org";
     contact.get().phoneNumber = phoneNumber.get();
     contact.get().postalAddress = address.get();
-ALLOW_DEPRECATED_DECLARATIONS_BEGIN
+    ALLOW_DEPRECATED_DECLARATIONS_BEGIN
```

Comment:
This indentation should probably not be changed.

Reply:
Grrr! Xcode keeps doing that to me! I've changed that in every single PR where I've touched this file.